<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Png2Dae GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="selectImageButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Select image</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageViewLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>211</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="optionsViewFrame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>251</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="zAxisScaleEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="zAxisScaleLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set z-axis scale factor, which affects the height of output model (default = &lt;span style=&quot; font-family:'monospace';&quot;&gt;0.25&lt;/span&gt;)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Z axis scale coeff</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="dimensionScaleLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set xyz-axis scale factor, which affect the dimensions of output model (default = &lt;span style=&quot; font-family:'monospace';&quot;&gt;0.05&lt;/span&gt;)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>XYZ axis scale coeff</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dimensionScaleEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="xAxisRotateEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="xAxisRotateLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set x-axis rotate angle (default = -90 degrees)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>X axis rotate angle</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="grayscaleOptionCheckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Convert arbitrary image to grayscale&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Grayscale</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="colorInverseOptionCheckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Using preliminary image color inverse&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Color inverse</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="stlToDaeSelect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>96</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Convert stl file model to dae model&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Stl to Dae</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="pngToStlSelect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>96</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Convert png file model to dae model&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Png to Stl</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="pngToDaeSelect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>96</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Convert png image to dae model&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Png to Dae</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="smoothingOptionCheckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>82</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Smoothing </string>
     </property>
    </widget>
    <widget class="QCheckBox" name="sharpnessOptionCheckbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>82</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Sharpness</string>
     </property>
    </widget>
    <widget class="QPushButton" name="textureImageSelect">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Select texture</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="outputFilenameEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="outputFilenameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Output file name:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="showImageWindowCheckbox">
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>10</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show the image windows</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="convertStartButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert!</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="convertStatusProgressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="value">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="outputFolderPathLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Output folder path</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="outputFolderPathEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="outputFolderSelect">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>!</string>
    </property>
   </widget>
   <zorder>optionsViewFrame</zorder>
   <zorder>selectImageButton</zorder>
   <zorder>imageViewLabel</zorder>
   <zorder>outputFilenameEdit</zorder>
   <zorder>outputFilenameLabel</zorder>
   <zorder>showImageWindowCheckbox</zorder>
   <zorder>convertStartButton</zorder>
   <zorder>convertStatusProgressBar</zorder>
   <zorder>outputFolderPathLabel</zorder>
   <zorder>outputFolderPathEdit</zorder>
   <zorder>outputFolderSelect</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
